enhancements/sig-architecture/225-placement-score-visibility/README.md

```diff
@@ -0,0 +1,173 @@
+# Placement Score Visibility
+
+## Release Signoff Checklist
+
+- [] Enhancement is `provisional`
+- [] Design details are appropriately documented from clear requirements
+- [] Test plan is defined
+- [] Graduation criteria for dev preview, tech preview, GA
+- [] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+This proposal extends existing Placement mechanisms to surface the scores of each cluster
+in PlacementDecision so that users (or future OCM features) can make advanced scheduling decisions
+based on the scored state of clusters.
+
+In addition, it adds support for different sort strategies on the decisions generated by a Placement.
+By default, decisions will continue to be sorted by "Name". An additional sort strategy "Score" will
+enable sorting decisions numerically in descending order by prioritizer score.
+
+## Motivation
+
+Placement scores are a powerful feature of OCM for scheduling workloads, but there is currently
+no reliable way for users to see what scores were assigned to clusters for a placement. Scores are
+reported in events, but these are ephemeral. Since PlacementDecisions are orderd alphanumerically
+there is no good way to see the relative scores of clusters in a decision other than recomputing
+them for each cluster in the decision group(s).
+
+A user may wish to use Placement to find all clusters that match some conditions, but then pick
+only the top N clusters for their workload. While this could be accomplished by setting
+`numberOfClusters: N` on the placement, this means a new placement must be created for every value
+of N. Another use-case would be to distribute workloads out to selected clusters randomly weighted
+by their scores. There is currently no way to do this in OCM without recomputing scores for each cluster
+listed in the PlacementDecisions. Displaying scores in an easy to consume manner gives more flexibility
+to the user, and enables a single Placement to be reused for many different scheduling decisions
+(reducing API pressure on the hub).
+
+## Proposal
+
+Add `SortBy` to `PlacementSpec` to configure the sort order for PlacementDecision. This field
```

Comment:
one of the issue with sorting by score is that if the score changed to frequently, the decision result will be updated frequently also. We need to think about some approach to avoid that, or at least some document to notice users.